azure-stack/operator/azure-stack-manage-portals.md

```diff
@@ -86,6 +86,9 @@ If you click the help icon (question mark) in the upper-right corner of the admi
 
 If you select **Help and support** (the question mark) in the upper right corner of the administrator portal, and then select **Azure roadmap**, a new browser tab opens and takes you to the Azure roadmap. By typing **Azure Stack Hub** in the **Products** search box, you can see all Azure Stack Hub roadmap updates.
 
+### Recommended browsers
```

Comment:
Same feedback as Matt. Remove /en-us from the URL to allow localization to work correctly.  I'd add a comma after recommend browsers.